plugins/cliconf/iosxr.py

```diff
@@ -29,6 +29,23 @@
   CLI commands from Cisco IOS XR network devices.
 version_added: 1.0.0
 options:
+  confirmed:
+    type: boolean
+    default: false
+    description:
+    - enable or disable commit confirmed mode
+    env:
+    - name: ANSIBLE_IOSXR_CONFIRMED
+    vars:
+    - name: ansible_iosxr_confirmed
+  timeout:
```

Comment:
Does this have any effect is `confirmed: False` or not set?

Reply:
when confirmed is false it will act as normal commit

Reply:
I meant, does timeout have any effect if `confirmed: False`?

Reply:
If not, we should see if we can add an argspec modifier here or a warning, that say `timeout` has effect when `confirmed` is not set.

Reply:
Maybe naming it `commit_confirmed_timeout` makes more sense here?

Reply:
Agreed